assets/variant-list.css

```diff
@@ -45,10 +45,10 @@ variant-list .quantity__button {
     overflow-y: auto;
   }
 
-  .variant-item__quantity-wrapper--no-info {
+  .variant-item__quantity-wrapper--no-info,
+  .variant-item__error {
     padding-left: 4.9rem;
   }
```

Comment:
Something ive noted to is that since the generic error is at the bottom, on mobile you may not see it (desktop is sticky). but im guessing this is an issue for cart too? 🤔 
https://screenshot.click/04-52-y26dw-v6wgh.mp4

Reply:
Good point, maybe we'll need to revisit the usefulness of showing a generic error at the bottom of the variant list (and in the cart). Especially in cases where the list is long.

Reply:
I think it's not an issue when you are on desktop (sticky bottom), but mobile and mobile _and_ desktop on cart have this issue 🤔 

cc @edmund-teh 

https://github.com/Shopify/dawn/issues/2767